### DISCUSSION TITLE
Option to Display Templated Pages in sidebar

### DISCUSSION THREAD

**Discussion by archiewood** [evidence-dev/evidence#2415] [General]
## Feature Description

View all the possible values of a templated page in the sidebar if they are at the top or second level, as per normal pages.

Suggest this would be configured as using the frontmatter option for the templated page

```
---
sidebar_link: true
---
```

I think the correct default behaviour would be for this to be true by default, but this would be a breaking change, so false by default would also work

It may be that this is a `prod only` feature

## Goal of Feature

Easier navigation through templated pages

## Current Solution / Workarounds

Set up nav structures within the main page itself

**Comment by archiewood** [comment]
(just came up in a call with a prospect)

**Comment by csjh** [comment]
Requires custom adapter for general solution. Could also be done (with extra user input, but works on dev mode) by allowing users to specify a subquery to [generate the pages with](https://kit.svelte.dev/docs/page-options#entries).

**Comment by archiewood** [comment]
templated page UX is pretty hard atm so I would be open to changing how you create them
